proto/pdpb.proto

```diff
@@ -815,6 +823,70 @@ message UpdateServiceGCSafePointResponse {
     uint64 min_safe_point = 4;
 }
 
+message GetGCSafePointV2Request {
+    RequestHeader header = 1;
+
+    uint32 keyspace_id = 2;
+}
+
+message GetGCSafePointV2Response {
+    ResponseHeader header = 1;
+
+    uint64 safe_point = 2;
+}
+
+message WatchGCSafePointV2Request {
+    RequestHeader header = 1;
+}
+
+// SafePointEvent is for the rpc WatchGCSafePointV2.
+message SafePointEvent {
+    uint32 keyspace_id = 1;
+    uint64 safe_point = 2;
+    EventType type = 3;
```

Comment:
What's the semantics of the `EventType` used here?

Reply:
When using the etcd watch interface, it represents the put/delete operation on the etcd path.